packages/client/src/components/TopCastDisplay/TopCastDisplay.jsx

```diff
@@ -17,6 +17,7 @@ const TopCastDisplay = ({ movieID }) => {
         position: 'absolute',
         right: 280,
         left: 0,
+        top: 1300,
```

Comment:
hmm, are we considering responsiveness in the project? Normally absolute positioning things is really ok, but it is safer in "small scale", so when I see large numbers I get a bit cautious :)

Reply:
This is done temporarily to adjust position of the card on the Movie Details Page.
Our team will be working again on the positions of the components.